lib/xboxrt/vcruntime/vcruntime_typeinfo.h

```diff
@@ -12,8 +12,8 @@
 
 struct __std_type_info_data
 {
-    char const *_UndecoratedName;
-    char const _DecoratedName[1];
+    char *_UndecoratedName;
+    char _DecoratedName[1];
```

Comment:
If we demangle at runtime, how will we know when to do it?
Wouldn't it have made more sense to keep it `const` and deal with it when implementing demangling?

Reply:
> If we demangle at runtime, how will we know when to do it?
MS demangles the name when `type_info::name()` is called and `_UndecoratedName` is `NULL`. It allocates a buffer for the string, demangles, and puts the address of the string in `_UndecoratedName`.

> Wouldn't it have made more sense to keep it const and deal with it when implementing demangling?
I don't think there's a significant benefit in keeping or removing `const` at this moment - it just bothered me because it was incorrect.